Introduce the User expressions group (#4985)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/expression.rst

   # All the above return: My country is France (FR)

.. _user_expressions_functions:

User Expressions |312|
----------------------

This group contains the expressions saved as **user expressions** using
the |fileSave| :sup:`Add the current expression to the user expressions` button.

> They are meant to be some important expressions you want to store and have a

Reviewer: ```suggestion
It is meant for important expressions you want to have
```

> quick access to. They are saved under the user profile and available in any

Reviewer: ```suggestion
quick access to. They are saved under the user profile and available in all
```

> expression dialog inside any project of the current user profile.

Reviewer: ```suggestion
expression dialogs inside all projects of the current user profile.
```
